CONTRIBUTING.md

```diff
@@ -12,9 +12,31 @@ Building Hubble.gl locally from the source requires node.js `>=10`.
 We use [yarn](https://yarnpkg.com/en/docs/install) to manage the dependencies.
 
 ```bash
+# Clone your fork of the hubble.gl repository
+git clone git git@github.com:<github username>/hubble.gl.git
+
+# Move to the directory where you put hubble.gl
+cd hubble.gl
+
+# Add the main repository as an upstream remote
+git remote add upstream "git@github.com:uber/hubble.gl.git"
+
+# Change to the master branch if not already on it (starts on master by default)
 git checkout master
+
+# Installs JavaScript dependencies
+yarn
 yarn bootstrap
+
+# Tests to make sure install went smoothly
 yarn test
+
+# Hubble.gl is intended for use with mapbox https://www.mapbox.com/
+export MapboxAccessToken=<your_access_token_here>
+
+# Start up hubble.gl example project
+# Optional: cd to a directory within the examples folder
+yarn start-local
```

Comment:
I think once in an example folder they would need to run `yarn` again before `yarn start-local`. Can you confirm?

Reply:
Confirmed. Examples would not run unless I installed dependencies again per directory. Fixed.